### DISCUSSION TITLE
Sorting or reordering function

### DISCUSSION THREAD

**Discussion by preacha** [SteamDeckHomebrew/decky-loader#175] [Ideas]
Currently, the plugins are displayed in the order in which they were installed. It would be great if you could change the order after installation.

**Comment by PartyWumpus** [comment]
(screenshot from decky dev in the discord server)
![Screenshot_20220915-082745](https://user-images.githubusercontent.com/48649272/190342723-979b3c8d-9a42-4fe7-b6d5-bdd36a5e1e7c.png)
